src/app/game/resources/resources.component.ts

```diff
@@ -1,22 +1,26 @@
-import { HttpClient } from '@angular/common/http';
 import { Component, OnInit } from '@angular/core';
+import { ResourcesService } from './resources.service';
 
 @Component({
   selector: 'app-resources',
   templateUrl: './resources.component.html',
   styleUrls: ['./resources.component.css']
 })
 export class ResourcesComponent implements OnInit {
+  constructor(private resourceService: ResourcesService) {}
+  food;
+  money;
 
-
-  constructor(private _http: HttpClient) {
+  ngOnInit() {
+    this.showResources();
   }
 
-  ngOnInit() {
-    this._http.get('https://api.myjson.com/bins/7xq2x1').subscribe(() => {
-      console.log('Http Call is success from component');
-    }, (error) => {
-      console.log('Http Call is failed from component');
-    });
+  showResources() {
+    this.resourceService
+      .getResources()
+      .subscribe(response => (this.food = response.resources[0].amount));
+    this.resourceService
+      .getResources()
+      .subscribe(response => (this.money = response.resources2[0].amount));
```

Comment:
@SamHerrera88 code duplication, and it's not even going to work, you won't have `resources2` in the response